package.json

```diff
@@ -9,7 +9,7 @@
     "pre-commit": "lint-staged",
     "prepare": "node prepare.js",
     "start:test-server": "node -r dotenv/config --experimental-json-modules e2e/test-server.js",
-    "test:playwright": "set NODE_OPTIONS=--experimental-vm-modules && npx playwright test"
+    "test:playwright": "SET NODE_OPTIONS=--experimental-vm-modules -r dotenv/config --experimental-json-modules && npx playwright test"
```

Comment:
`SET` doesn't work, can you check this part?

Reply:
Error: Cannot find module '/apihub/public/temp/seed-credentials.json' imported from '/apihub/e2e/routes/seeds/generated-credentials.test.js'